node/src/runtime/mainnet.rs

```diff
@@ -201,27 +223,39 @@ pub fn testnet_genesis(
 				session_time_in_seconds: 4 * 3600, // 4 hours
 			},
 		},
-		assets: Default::default(),
+		assets: AssetsConfig {
+			assets: vec![
+				// As per discussion with Pavel, we don't want to commit ourselves to any assets at genesis.
+				// But we will add expected assets for our cosmos testnet.
+
+				// id, owner, is_sufficient, min_balance
+				(1, multisig.clone(), true, 1),
+				(2, multisig.clone(), true, 1),
+			],
+			metadata: vec![
+				(
+					1,
+					"GGx Cosmos testnet stake token".into(),
+					"STAKE".into(),
+					18,
+				),
+				(2, "GGx Cosmos testnet ert token".into(), "ERT".into(), 18),
+			],
+			accounts: vec![],
+		},
 		vesting: Default::default(),
 		indices: Default::default(),
 		im_online: Default::default(),
 		society: Default::default(),
 		currency_manager: CurrencyManagerConfig {},
 		account_filter: AccountFilterConfig {
-			allowed_accounts: initial_authorities
-				.iter()
-				.map(|x| (x.id.clone(), ()))
-				.collect(),
+			allowed_accounts: vec![(validator_identity.id, ())],
 		},
 		ics_20_transfer: Ics20TransferConfig {
-			asset_id_by_name: vec![("ERT".to_string(), 666)],
+			asset_id_by_name: vec![("ERT".to_string(), 1), ("stake".to_string(), 2)],
 		},
 		eth_2_client: Eth2ClientConfig {
 			networks: vec![
-				(
-					webb_proposals::TypedChainId::Evm(1),
-					NetworkConfig::new(&Network::Mainnet),
-				),
 				(
 					webb_proposals::TypedChainId::Evm(5),
 					NetworkConfig::new(&Network::Goerli),
```

Comment:
@akorchyn Is Goerli still in use?

Reply:
the Goerli network was decided to come to an end and will be deprecated in April 2024, which will completely cease the support for the network.

Reply:
But it's still running though